frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/InstallNodeAndNpmMojo.java

```diff
@@ -73,40 +79,38 @@ protected boolean skipExecution() {
 
     @Override
     public void execute(FrontendPluginFactory factory) throws InstallationException {
-        ProxyConfig proxyConfig = MojoUtils.getProxyConfig(session, decrypter);
+        ProxyConfig proxyConfig = MojoUtils.getProxyConfig(this.session, this.decrypter);
         String nodeDownloadRoot = getNodeDownloadRoot();
         String npmDownloadRoot = getNpmDownloadRoot();
-        Server server = MojoUtils.decryptServer(serverId, session, decrypter);
+        Server server = MojoUtils.decryptServer(this.serverId, this.session, this.decrypter);
         if (null != server) {
-            factory.getNodeAndNPMInstaller(proxyConfig)
-                .setNodeVersion(nodeVersion)
-                .setNodeDownloadRoot(nodeDownloadRoot)
-                .setNpmVersion(npmVersion)
-                .setNpmDownloadRoot(npmDownloadRoot)
-                .setUserName(server.getUsername())
-                .setPassword(server.getPassword())
-                .install();
+            factory.getNodeInstaller(proxyConfig).setNodeVersion(this.nodeVersion)
```

Comment:
It has just been changed due to IDE auto-formatting reasons. Basically, I was lazy to format code by myself. However, I would guess that you have your own favorite code style, which is not shared anyhow? I am applying my code styles while merging before doing a review. Thus, I can see what actually has been changed and merge the code with the correct code style.